docs/01-design-doc.md

```diff
@@ -0,0 +1,31 @@
+# Newsletters RoundUp Page Design Document
+
+## Context and Scope
+
+The [newsletters round up page][] is currently underwhelming and doesn't do the best job of advertising our newsletters, reflecting the quality of the product or providing a good user experience. In the process of revamping this page, we should look at using more modern technologies for developing and rendering the page. In particular, we want to bring the page in line with other modern Guardian projects by using React.
+
+## Goals and non-Goals
+
+-   Improve developer experience by using the right tool for the job (a frontend technology in this case).
```

Comment:
Nitpicking but I'd possibly put this third in the list - not that developer experience isn't important, but I think it's not the main motivator for this work.

I also think that it would be good to justify _why_ we think the current page is not using the right tool. To my mind a good justification is that `frontend` as a project is moving away from rendering more generally, and so it makes sense for this to live somewhere else, and so there is benefit to being consistent with other rendering platforms in terms of technology choices if we have to build it again anyway.

Reply:
I hadn't considered this to be an ordered list, but I've shifted it around to reflect this because the order will suggest priority and importance as you've pointed out.

As for the point about why it's not the right tool, there's a line in the `Context and Scope` section about what you've mentioned here, but I'll flesh it out and give more context to it. Ta